styles.css

```diff
@@ -271,14 +278,16 @@
   cursor: pointer;
 }
 
-#about .about-para button:hover {
+#about .about-des .about-para button:hover {
   scale: 0.8;
 }
 
 #about .experience {
   display: flex;
   flex-direction: column;
   align-items: center;
+  width: 100%;
+  height: 100%;
   margin-top: 3rem;
 }
```

Comment:
[OPTIONAL] Your experience card looks a little bit different from the given design😉 I guess the previous reviewers didn't take note of it or you changed it, but it is a requirement that you always stick to the given design🤗 I suggest you make use of `gap` instead of ` justify-content: space-around;` as you have used🤗

your_card
<img width="896" alt="YOUR____CARDS" src="https://user-images.githubusercontent.com/64643995/213386517-783acafe-c8d3-42bf-be80-50e7063c282e.png">

design
<img width="460" alt="DESIGN CARD" src="https://user-images.githubusercontent.com/64643995/213386603-8c38828a-cfa1-4f91-bb57-ba21668f4e36.png">